docs/dev/style-guide.rst

```diff
@@ -0,0 +1,53 @@
+Documentation Style Guide
+=========================
+
+This document will serve as the canonical place to define how we write documentation at Read the Docs.
+The goal is to have a shared understanding of how things are done,
+and document the conventions that we follow.
+
+Let us know if you have any questions or something isn't clear.
+
+The brand
+---------
+
+We are called **Read the Docs**.
+The ``the`` is not capitalized.
+
+We do however use the acronym **RTD**.
+
+Titles
+------
+
+For page titles we use sentence case.
+This means only proper nouns and the first word are capitalized::
+
+# Good ✅
+How we handle support on Read the Docs.
+
+# Bad 🔴
+How we Handle Support on Read the Docs
+
+If the page includes multiple sub-headings (H2, H3),
+we use sentence case there as well.
+
+Content
+-------
+
+* Use ``:guilabel:`` when referring to an element on the screen.
```

Comment:
```suggestion
* Use ``:menuitem:`` when referring to an item in navigation or the exact text of a link label.
* Use ``:guilabel:`` when referring to an visual element on the screen - such as a button, drop down or input field.
* Use ``**bold text**`` when referring to a non-interactive text element, such as a header.
```

Does this reflect what we want to do?

Reply:
What is `:menuitem:`? I've never seen it and I'm not finding it in the docs.

Reply:
typo - it's this one: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection

Reply:
I didn't know about this role! I have used a string of `guilabel` roles, which is more tedious. Assuming the display output is as nice, this seems like a really handy pattern :+1:

Reply:
Yea, I just found `menuselection` the other day and thought that was way nicer.